APPLICATION_PROGRESS.md

```diff
@@ -400,6 +400,9 @@ Will move from Apps repo to Libs repo soon
 
 <h3>Oracle Node</h3>
 
+- Feature complete, but could most likely be improved once event monitoring is added to
```

Comment:
Not in this implementation no.  I originally thought it would from my initial [research](https://docs.chain.link/docs/architecture-request-model/#off-chain-oracle-node)) into oracles, but this is an extremely simple oracle.  Maybe this line should be removed from the readme or say "This oracle could be improved to resemble something closer to chainlink once event monitoring is added"